cookie-banner-rules-list.json

```diff
@@ -636,6 +636,265 @@
       },
       "id": "c5243e7c-eb86-4a9d-947c-7129e99fbd72",
       "last_modified": 1660836869997
-    }
+    },
+    	{
+  "click": {
+    "optIn": "button#onetrust-accept-btn-handler",
+    "optOut": "button#onetrust-reject-all-handler",
+    "presence": "div#onetrust-consent-sdk"
+  },
+  "domain": "cloudflare.com",
+  "schema": 1665622072973,
+  "cookies": {},
+  "id": "0b42c238-b54d-4106-8d9a-81df5bc0b3ae"
+  },
+    {
+  "click": {
+    "optIn": "button#L2AGLb",
+    "optOut": "button#W0wltc",
+    "presence": "div#CXQnmb"
+  },
+  "domain": "google.com.hk",
+  "schema": 1665664785245,
+  "cookies": {
+    "optIn": [
+      {
+        "name": "SOCS",
+        "value": "CAISHAgBEhJnd3NfMjAyMjEwMDYtMF9SQzMaAnJvIAEaBgiAqp2aBg",
+        "isSecure": true
+      }
+    ],
+    "optOut": [
+      {
+        "name": "SOCS",
+        "value": "CAESHAgBEhJnd3NfMjAyMjEwMDYtMF9SQzMaAnJvIAEaBgiAqp2aBg",
+        "isSecure": true
+      }
+    ]
+  },
+  "id": "4a947309-9a90-45bc-916c-b76bcd61bed1"
+  },
+    {
+  "click": {
+    "optIn": "button#onetrust-accept-btn-handler",
+    "optOut": "button#onetrust-reject-all-handler",
+    "presence": "div#onetrust-consent-sdk"
+  },
+  "domain": "msn.com",
+  "schema": 1665665451546,
+  "cookies": {
+    "optIn": [
+      {
+        "name": "eupubconsent-v2",
+        "value": "CPg0HlKPg0HlKAcABBENCFCsAP_AAEPAACiQImtf_X__b3_j-_5_f_t0eY1P9_7_v-0zjhedt-8N3d_X_L8X52M7vF36pq4KuR4Eu3LBAQdlHOHcTUmw6IkVqzPsbk2cr7NKJ7PEmnMbO2dYGH9_n93TuZKY7______z_v-v_v____f__-3_3__5_1---wAAB8gdzLv9____393P___9v-_9_____-CIYBJhqXkAXZljgybRpVCiBGFYSHQCgAooBhaIrCB1cFOyuAn1BCwAQCpCcCIEGIKMGAQACCQBIREBIAeCARAEQCAAEAKsBCAAjYBBYAWBgEAAoBoWIEUAQgSEGRwVHKYEBUi0UE9lYglB3saYQhlnARQIAAABABrNECwMhIWDmOAJAS8WSB5ihfIBAaBCAFYALgAhgBkADLAGyAOwAfgBAACCgEYAKeAVeAtAC0gGsAN4AdUA-QCHQEVAJEATYAnYBSIC5AGEgMPAYwAycBnIDPAGfAPwDACgBzAHUASEAkUBkYDdAHEgOzAe6BD4QALABIAOYA3gCQgEigN0AcSA7MB7oD7AIfCIDQAVgBDADIAGWANkAdgA_ACAAEYAKeAVcA1gB1QD5AIdASIAmwBOwCkQFyAMJAYeAycBnIDPgH4CoDgAFAAhgBMAC4AI4AZYA7AB-AEYAI4AVeAtAC0gG8ASCAmIBNgCmwFsALkAXmAw8BkQDOQGeAM-AbkA_ACF4oAaANoAcwA8ACCgHVAR6AkUBrwDbwHEgPsAgeBBsZAaAAoAEMAJgAjgBlgDsgH2AfgBGACOAFXAK2AbwBJwCYgE2ALRAWwAvMBh4DIgGcgM8AZ8A-IB-AELxgAwAbQA5gB4AFiAOqAj0BIoC8gG3gOJAfYBBsAA.f_gACHgAAAAA"
```

Comment:
Curious how stable this is. Since we're in Nightly-only mode for now we can land it and remove it again as needed.